package.json

```diff
@@ -17,7 +17,7 @@
     "lint-staged": "lint-staged",
     "pretty": "prettier --write .",
     "prettier:check": "prettier --check .",
-    "test": "lerna run test --no-private",
+    "test": "lerna run test --no-private --stream",
```

Comment:
What does this do here?

Reply:
https://github.com/lerna/lerna/blob/main/commands/run/README.md#--stream

<img width="1024" alt="image" src="https://user-images.githubusercontent.com/41187899/197783702-cf94c537-1632-414d-b2a3-b98f82b23122.png">

It's an option I added on Marc's advice. It adds extra outputs, without it the CI only out put was "exited with code 1".